src/Mod/TechDraw/Gui/QGIViewBalloon.cpp

```diff
@@ -173,8 +173,8 @@ void QGIViewBalloon::onAttachPointPicked(QGIView *view, QPointF pos)
 
     if (balloon->OriginIsSet.getValue() == false) {
         /* Move origin by half of cursor size */
```

Comment:
Could you remove this comment ?
BTW, your solution is much cleaner. Thanks.

Reply:
OK, removed....

Reply:
I might be a bit picky here but you should have squashed this commit with you previous one.

Reply:
What do you mean? If I go to "Files changed" on the top of this page, everything is there. Am I missing something?

Reply:
In your local git branch you can merge your two commits (rebase -i). Then you force push the resulting commit. Your pull request will be updated automatically.

Reply:
Ok, WandererFan already merged it to master, but I will remember it for next time.
Git is very confusing....

Reply:
No problem.